ansible/roles_ocp_workloads/ocp4_workload_ansible_automation_platform/defaults/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+become_override: false
+ocp_username: user-redhat.com
+silent: false
+tmp_dir: /tmp/{{ guid }}
+tmp_kubeconfig: "{{ tmp_dir }}/.kube/config"
+
+automation_controller_project: "automation-controller"
+automation_controller_project_display: "automation-controller"
+# automation_controller_hostname:  # LEAVE EMPTY AS IT'S CALCULATED IN THE ROLE
+automation_controller_app_name: "automation-controller"
+automation_controller_admin_secret: "automation-controller-admin-password"
+automation_controller_admin_password: "r3dh4t1!"
```

Comment:
Please no `r3dh4t1!` password. Better to throw an error because the variable was not set.